gallery/src/styles/NxFormLayout/NxFormHorizontalLayoutExample.tsx

```diff
@@ -4,53 +4,87 @@
  * the terms of the Eclipse Public License 2.0 which accompanies this
  * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
  */
-import React from 'react';
+import React, { useState } from 'react';
 
 import { NxCheckbox } from '@sonatype/react-shared-components';
 import { NxRadio } from '@sonatype/react-shared-components';
-
-import { NxFontAwesomeIcon } from '@sonatype/react-shared-components';
-import { faCalendar } from '@fortawesome/free-solid-svg-icons';
+import { NxButton } from '@sonatype/react-shared-components';
+import { NxStatefulTextInput } from '@sonatype/react-shared-components';
 
 export default function NxFormLayoutExample() {
+  function validator(val: string) {
+    return val.length ? null : 'Must be non-empty';
+  }
+
+  const [isRed, setIsRed] = useState(false),
+  [isBlue, setIsBlue] = useState(false),
+  [isGreen, setIsGreen] = useState(false),
+  toggleRed = () => setIsRed(!isRed),
+  toggleBlue = () => setIsBlue(!isBlue),
+  toggleGreen = () => setIsGreen(!isGreen);
+
+  const [color, setColor] = useState<string | null>(null);
+
+  // function onSubmit(evt) {
+  //   evt.preventDefaul();
+  //   alert('Submitted!');
+  // }
```

Comment:
should this be removed?

Reply:
> should this be removed?

So Ross asked that this be added here: https://github.com/sonatype/sonatype-react-shared-components/pull/122#discussion_r470777647

But when I did it wouldn't compile, so if you could hep me figure out why that would be cool.

Reply:
> > should this be removed?
> 
> So Ross asked that this be added here: [#122 (comment)](https://github.com/sonatype/sonatype-react-shared-components/pull/122#discussion_r470777647)
> 
> But when I did it wouldn't compile, so if you could hep me figure out why that would be cool.

As discussed removed entirely here: https://github.com/sonatype/sonatype-react-shared-components/pull/122/commits/885306b2ba998cb4d78a4f4b5074bad813adc2d7